### PR TITLE
cloud: add apikey references

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -286,7 +286,9 @@
 - API
   - [API Overview](/tidb-cloud/api-overview.md)
   - API Reference
-    - [v1beta1](https://docs.pingcap.com/tidbcloud/api/v1beta1)
+    - v1beta1
+      - [API Key](https://docs.pingcap.com/tidbcloud/api/v1beta1/apikey)
+      - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1/billing)
```
Comment (ran-huang) [inline TOC-tidb-cloud.md:291]:
```suggestion
      - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1/billing)
      - [MSP](https://docs.pingcap.com/tidbcloud/api/msp/v1beta1)
```

File: tidb-cloud/api-overview.md [hunk 2]
```diff
@@ -23,6 +23,8 @@ To start using TiDB Cloud API, refer to the following resources in TiDB Cloud AP
 - [Authentication](https://docs.pingcap.com/tidbcloud/api/v1beta#section/Authentication)
 - [Rate Limiting](https://docs.pingcap.com/tidbcloud/api/v1beta#section/Rate-Limiting)
 - API Full References
-    - [v1beta1](https://docs.pingcap.com/tidbcloud/api/v1beta1)
+    - v1beta1
+        - [API Key](https://docs.pingcap.com/tidbcloud/api/v1beta1/apikey)
+        - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1/billing)
```
Comment (ran-huang) [inline tidb-cloud/api-overview.md:28]:
```suggestion
        - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1/billing)
        - [MSP](https://docs.pingcap.com/tidbcloud/api/msp/v1beta1)
```

File: tidb-cloud/api-overview.md [hunk 3]
```diff
@@ -23,6 +23,9 @@ To start using TiDB Cloud API, refer to the following resources in TiDB Cloud AP
...
```
Comment (hfxsd) [inline tidb-cloud/api-overview.md:28]:
```suggestion
        - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1#tag/Billing)
```

File: TOC-tidb-cloud.md [hunk 4]
```diff
@@ -286,7 +286,10 @@
...
```
Comment (hfxsd) [inline TOC-tidb-cloud.md:291]: API Key 的链接没找到
```suggestion
      - [API Key](https://docs.pingcap.com/tidbcloud/api/v1beta1/apikey)
      - [Billing](https://docs.pingcap.com/tidbcloud/api/v1beta1#tag/Billing)
```

**Review by hfxsd — APPROVED** [review]

**Pull request merged**